src/app/payment/paymentMethod/MasterpassPaymentMethod.tsx

```diff
@@ -20,8 +20,17 @@ const MasterpassPaymentMethod: FunctionComponent<MasterpassPaymentMethodProps &
         },
     }), [initializePayment]);
 
+    const formatLocale = useCallback((localeLanguage: string) => {
+        const auxLocale = localeLanguage.replace(/[-_]/g, '_');
+        const formatedLocale = auxLocale.includes('_') ? `${auxLocale}` : `${auxLocale}_${auxLocale}`;
+
+        return formatedLocale.toLowerCase();
+    }, []);
```

Comment:
This method is react agnostic and uses nothing from the component scope. Could we move it outside the component?